src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
The upcoming [PSR](https://github.com/phpDocumentor/fig-standards/blob/6861e37b4e8d2bc34af980e2e9457ee68816515d/proposed/phpdoc.md#keyword) (and phpdocumentor) allows both, I think using full type name is more straightforward.